### DISCUSSION TITLE
Typescript types

### DISCUSSION THREAD

**Discussion by moonraker22** [creativelifeform/three-nebula#248] [General] [answered]
For anyone looking for types I created types for the library and got them merged into Definitely Typed so you can install types with 
```
npm i @types/three-nebula
```
here's the npm page [https://www.npmjs.com/package/@types/three-nebula](https://www.npmjs.com/package/@types/three-nebula)

**Comment by rohan-deshpande** [comment, marked as answer]
The library is now natively written in TypeScript, thanks for your efforts!
